src/query/sql/src/planner/binder/table.rs

```diff
@@ -310,7 +322,21 @@ impl<'a> Binder {
                 },
             };
             bind_context.add_column_binding(column_binding);
+            column_indexs.insert(column.index());
+            if !has_path_indices {
+                // create table t (c0 tuple(int, int), c1 string null) in planner col index is : (0: c0, 1: c1, 2: c0:2 3: c0:2)
+                // in storage column (0: PrimitiveColumn, 1: PrimitiveColumn, 2: NullableColumn)
+                has_path_indices = column.has_path_indices();
+            }
+            let col_stat = table
+                .table()
+                .column_statistics_provider()
+                .await?
+                .column_statistics(column.index() as ColumnId);
+            col_stats.insert(column.index(), col_stat);
```

Comment:
`ColumnIndex` in `Metadata` is a global id, it is different from `ColumnId` in parquet. We need a transformation function to do the mapping.